content/guides/user-guides/csi/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: CSI
```

Comment:
This content will come under UserGuide -> NBP -> CSI

Reply:
corrected @asifdxtreme